otel/otel-configuration-of-collector.adoc

```diff
@@ -0,0 +1,15 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="otel-configuration-of-collector"]
+= Configuration of the OpenTelemetry Collector of the {OTELShortName}
+include::_attributes/common-attributes.adoc[]
+:context: otel-configuration-of-collector
+
+toc::[]
+
+The {OTELName} Operator uses a custom resource definition (CRD) file that defines the architecture and configuration settings to be used when creating and deploying the {OTELShortName} resources. You can install the default configuration or modify the file.
```

Comment:
Nit - this abstract doesn't talk about the collector specifically. It might be good if it explained what it is, what it does, why you would want to set config options for it, etc.

Reply:
Thank you. I'll add your nit to the Jira that I created for the next release to add an intro into the product/architecture, which is currently missing (currently, it is on the first GA version).